### ISSUE TITLE
[FEATURE] Add contributing guidelines

### ISSUE THREAD

**Issue by Panquesito7** [WebXDAO/nextjs-dapp-template#7]
### Detailed description

We're currently missing contributing guidelines for this repository. I recommend adding contributing guidelines in each repository.

### Context

This will make the contribution process easier and will help our contributors understand better what the project is about and how can they contribute.

### Additional information

The readme also mentions a contributing file that is currently unavailable, thus, leading to an invalid link page.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mkubdev** [comment]
Yup, nice idea, we need to update them before the start of Hacktoberfest. 💡

**Comment by mkubdev** [comment]
@Panquesito7 Do you know where I can find an example?

**Comment by Panquesito7** [comment]
We could use this as a template. It is good, IMO: https://github.com/dev-protocol/devprotocol.xyz/blob/update_links/CONTRIBUTING.md

**Comment by mkubdev** [comment]
Awesome thanks!
